### PR TITLE
Added path to URL column

### PR DESCRIPTION
**Pull request by Splinter0**
#16

### CODE REVIEW

File: packages/frontend/src/components/dashboard/TemplateTable/Container.vue [hunk 1]
```diff
@@ -105,6 +105,20 @@ const isAnalyzing = computed(() => {
   return analysisService.jobState.type === "Analyzing";
 });
 
+const defaultPorts = [80, 443];
+
+const getURLColumnValue = (template: TemplateDTO) => {
+  let url = `${template.meta.method} ${template.meta.isTls ? 'https' : 'http'}://${ template.meta.host }`;
+  if (!defaultPorts.includes(template.meta.port)) {
+    url += `:${template.meta.port}`;
+  }
```
Comment (Corb3nik) [inline packages/frontend/src/components/dashboard/TemplateTable/Container.vue:114]: Should we take into account the scenario where `port === 80` and `isTls === true` or `port === 443` and `isTls === false`

In that case, `https://google.com/` would be wrong and we'd want to show `https://google.com:80` maybe?

Comment (Splinter0) [reply packages/frontend/src/components/dashboard/TemplateTable/Container.vue:114]: Sure! Small edge case but I think it's easy enough to implement. Will make the changes soon

Comment (Splinter0) [reply packages/frontend/src/components/dashboard/TemplateTable/Container.vue]: Updated!

**Comment by Corb3nik** [comment]
Looks good :)

**Pull request merged**